tests/helper/helper_dev.go

```diff
@@ -169,6 +170,12 @@ func (o DevSession) WaitSync() ([]byte, []byte, error) {
 	return outContents, errContents, nil
 }
 
+func (o DevSession) CheckNotSynced(timeout time.Duration) {
+	time.Sleep(timeout)
```

Comment:
How about using [`Consistently`](https://onsi.github.io/gomega/#consistently) to repeatedly check the assertion during this period of time? This way, I think it will basically fail fast if the matcher does not pass, rather than just waiting until the end.